app/views/planning_applications/_assessment_dashboard.html.erb

```diff
@@ -8,6 +8,13 @@
 
 <%= render "proposal_header" %>
 
+<div class="govuk-grid-column-two-thirds">
+  <% if @planning_application.validation_requests %>
+    <%= render "overdue_banner" unless overdue_requests(@planning_application.validation_requests).empty? %>
+    <%= render "response_banner" unless closed_requests(@planning_application.validation_requests).empty? %>
```

Comment:
minor tweak, but I prefer `if .present?` as it's not a double negative.

Reply:
Fixed!